### ISSUE TITLE
Atmos doesnt try and do pressure or temp damage to windows/walls on the tile beside it.

### ISSUE THREAD

**Issue by Jalleo** [yogstation13/Yogstation#9386]
This was tested on the latest version of Yog and is confirmed on NSV, Bee (NSV was recently updated so it has rather current bee code)

## Reproduction:
Make a TEG. Use plasma windows that face to the tiles you want to burn to keep it contained without any issues.
E.G.
![image](https://user-images.githubusercontent.com/4549053/89332879-0a9c4380-d68c-11ea-96e2-a26188142467.png)
Of which a setup like this when it gets released by a admin or by accident causes
![image](https://user-images.githubusercontent.com/4549053/89332950-230c5e00-d68c-11ea-87e5-d70e90ef517f.png)
There is no temp or pressure sharing to tiles outside of the fire so its technically perfectly contained and gets to temperatures hotter than anything that is actually possible in a lab even theoretically (A lot more than the sun)

This allows for TEGs on the NSV13 codebase after a 25% nerf to power generation get to 250 MW if setup correctly or even higher to the point where the gasses may overflow on the integers. That last part is currently unproven.

**Comment by Jalleo** [comment]
See the other issue report to a link for the TG PR that likely fixes most of the issues there.

**Comment by redmoogle** [comment]
We're not a tg downstream pls stop we never implemented this

**Comment by LemonInTheDark** [comment]
Unless monster removed it, you have super-conduction in your codebase. it's pre open source old

**Comment by LemonInTheDark** [comment]
this is the commit that broke it https://github.com/yogstation13/Yogstation/commit/ebf735184c5209fa3c03efefe9de0c72206802c1#diff-e7869e3e3569e5b2167dbacae56b8e6eR236, I can't tell if it's been fixed yet

**Comment by redmoogle** [comment]
iirc superconduction just relates to speedy heat transfer, and also windows will only start to break once the fire is in it

**Comment by LemonInTheDark** [comment]
yeah pressure is a poor way of putting it. windows should break just depending on heat but the support for that isn't in the codebase currently
